client/public/styles/auth.scss

```diff
@@ -0,0 +1,15 @@
+.auth-card-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.auth-card {
+  width: 400px;
```

Comment:
For this would it make sense to have the width also be variable (%) depending on resolution of screen